nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fnodetypes/Shor-UI_w1/NodeType.tosca

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/quantum/applications/nodetypes" id="quantumApplicationNodeTypes-Shor-UI_w1">
+    <NodeType name="Shor-UI_w1" abstract="no" final="no" targetNamespace="https://ust-quantil.github.io/quantum/applications/nodetypes" winery:bordercolor="#c9c203">
+        <DerivedFrom xmlns:otntyIgeneral="http://opentosca.org/nodetypes" typeRef="otntyIgeneral:NGINX-Application_w1"/>
+        <Interfaces>
+            <Interface name="http://opentosca.org/interfaces/connections">
+                <Operation name="connectTo">
+                    <InputParameters>
+                        <InputParameter name="VMIP" type="string" required="no"/>
```

Comment:
Does not match with the input parameters of the connectTo.sh file. There is the additional parameter ContainerIP. However, adding it here would most probable result in removing the operation in the plan due to missing parameters? So, I´m not sure how to fix this until we use different NodeTypeImpls depending on the underlying hardware...